spec/shared_examples_prison_slots_and_data.rb

```diff
@@ -0,0 +1,43 @@
+RSpec.shared_examples "slots and data" do
```

Comment:
Thank you.

Hmmm, it did at one point.  I must have cut it out by accident.  I'll put
it back in.

On 25 September 2015 at 14:57, Steve Marshall <notifications@github.com>
wrote:

> In spec/shared_examples_prison_slots_and_data.rb
> <https://github.com/ministryofjustice/prison-visits/pull/332#discussion_r40431570>
> :
>
> > @@ -0,0 +1,43 @@
> > +RSpec.shared_examples "slots and data" do
>
> This is a nice refactoring-out of these [image: :stars:]
>
> Might be nice if the name had prison in it, though?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/prison-visits/pull/332/files#r40431570>
> .
>